kolibri/core/auth/management/commands/sync.py

```diff
@@ -59,12 +62,17 @@ def add_arguments(self, parser):
         parser.add_argument(
             "--username",
             type=str,
-            help="username of superuser on server we are syncing with",
+            help="username of superuser or facility admin on server we are syncing with",
         )
         parser.add_argument(
             "--password",
             type=str,
-            help="password of superuser on server we are syncing with",
+            help="password of superuser or facility admin on server we are syncing with",
+        )
+        parser.add_argument(
+            "--user-id",
```

Comment:
For consistency with the `--facility` argument, could we change this just to `--user`?

Reply:
Main thing to disambiguate it from is `--username` (which is used for authentication, vs this one, which is specifying a) that it's a single-user sync, and b) which user to sync)

Reply:
![image](https://user-images.githubusercontent.com/618416/116146999-02f65200-a694-11eb-8767-ae2d9dd5130b.png)

Reply:
👍